prac_06/my_date.py

```diff
@@ -0,0 +1,75 @@
+"""
+Describe a date
+"""
+MONTH_29 = (2,)
+MONTH_30 = (4, 6, 9, 11)
+MONTH_31 = (1, 3, 5, 7, 8, 10, 12)
+
+
+def get_max_day(month: int, year: int):
+    if month in MONTH_29:
+        max_day = 29 if year % 4 == 0 else 28
```

Comment:
this leap year condition case needs more checking.
4 years a leap, 100 years no leap, 400 years a leap.